src/main/navigation/navigation.h

```diff
@@ -87,7 +87,10 @@ typedef enum {
 typedef enum {
     NAV_RTH_ALLOW_LANDING_NEVER = 0,
     NAV_RTH_ALLOW_LANDING_ALWAYS = 1,
-    NAV_RTH_ALLOW_LANDING_FS_ONLY = 2, // Allow landing only if RTH was triggered by failsafe
+    NAV_RTH_ALLOW_LANDING_APROACH = 2,
+    NAV_RTH_ALLOW_LANDING_FS_ONLY = 3,     // Allow landing only if RTH was triggered by failsafe
```

Comment:
@hali9 can we keep previously used IDs in place? So FS_ONLY stays at 2 and new modes go into new IDs?

Reply:
Yes, sure.